code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -58,9 +58,6 @@
 	user.last_action = world.time
 	user.newtonian_move(get_dir(A, user))
 	var/turf/T = get_turf(src)
-	if(reagents.has_reagent(/datum/reagent/toxin/acid))
-		message_admins("[ADMIN_LOOKUPFLW(user)] fired sulphuric acid from \a [src] at [ADMIN_VERBOSEJMP(T)].")
-		log_game("[key_name(user)] fired sulphuric acid from \a [src] at [AREACOORD(T)].")
```

Comment:
dont just remove it fully, remove only the message_admins(), its important to log this kind of stuff imo.

Reply:
Done!